### PR TITLE
Added Missing jobID

### PR DESCRIPTION
**Pull request by jumaallan**
Story: N/A

## Summary

Added missing jobId on SmartSelfieAuthenticationFragment and SmartSelfieEnrollment

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
 # Changelog
 
+## 10.0.0-beta11 (unreleased)
```
Comment (vanshg) [inline CHANGELOG.md:3]: Why create this section? we haven't released beta10 yet

Comment (jumaallan) [reply CHANGELOG.md:3]: I forgot to remove (unseleased) on beta10

We need to ship this fix for Umba. They are stuck

**Review by vanshg — APPROVED** [review]
